aisynphys/database/schema/patch_seq.py

```diff
@@ -49,6 +49,7 @@
     ('last_map', 'str', 'mapping from the last batch run', {'index': True}),
     ('last_score', 'str', 'mapping score from the last batch run', {'index': True}),
     ('mapped_subclass', 'str', 'Subclass that this cell mapped to', {'index': True}),
+    ('batch', 'str', 'Shiny batch number, used for tracking updates', {'index': True}),
```

Comment:
Consider putting stuff like this in `meta` since it is meaningless to external users. (It's ok to leave it here if you need to be able to query quickly from it)

Reply:
yeah I want to have access to it in MAZ so I can see what things look like/what was new in each batch....there is admittedly a lot of stuff in the patchseq table that is for tracking, should we consider making a special patchseq_meta table?

Reply:
Let's keep it all in the patchseq table for now; no big deal.